Philips.CodeAnalysis.MsTestAnalyzers/TestHasDescriptionAnalyzer.cs

```diff
@@ -20,7 +20,10 @@ public class TestHasDescriptionAnalyzer : TestMethodDiagnosticAnalyzer
 
 		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(Rule); } }
 
-		protected override TestMethodImplementation OnInitializeTestMethodAnalyzer(AnalyzerOptions options, Compilation compilation, MsTestAttributeDefinitions definitions) => new TestHasDescription(definitions);
+		protected override TestMethodImplementation OnInitializeTestMethodAnalyzer(AnalyzerOptions options, Compilation compilation, MsTestAttributeDefinitions definitions)

```

Comment:
Not sure if this is an improvement...

Reply:
This pertains to https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0022#csharp_style_expression_bodied_methods  In the past, we set "when_on_single_line" and enjoyed it initially.  However, 2 problems ensued: (1) Sometimes a method bounces between 1 line and 2 lines long. In those situations it's annoying to have to constantly flip back and forth between the 2 modes.  (2) When the single line is quite long, it's annoying that it's all on one physical line as it involves scrolling. So moving to multiple lines implies when not use a traditional block style anyway.

Reply:
@ynsehoornenborg will resolve and move forward but feel free to reach out as always.